python/foxglove-sdk/python/foxglove/__init__.py

```diff
@@ -86,6 +91,34 @@ def start_server(
             channel_filter=channel_filter,
         )
 
+    def start_cloud_sink(
```

Comment:
Why do we have this wrapping function rather than exporting the original and documenting the function in the stub file? (I realize the same question applies for the existing `start_server`.)